modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl

```diff
@@ -232,6 +234,10 @@ public class ${schemaName} <#if dtoParentClassName?has_content>extends ${dtoPare
 				description = "${propertySchema.description?j_string}"
 			</#if>
 		)
+		<#if propertySchema.jsonString>
+			@JsonDeserialize(using = JSONStringStdDeserializer.class)
+
```

Comment:
I think this extra line is not matching the pattern

Reply:
I think this annotation is not sorted. Maybe it should be included after `@JsonAnySetter`